tests/functional/u/unnecessary/unnecessary_dunder_call_py38.rc

```diff
@@ -1,3 +1,3 @@
 [testoptions]
-max_pyver=3.8
```

Comment:
Added in #9034, but I think the idea was to run this test on 3.8, not skip it on 3.8.

max_pyver is actually pretty confusingly named.